src/disjoint_mut.rs

```diff
@@ -465,13 +451,22 @@ impl From<RangeToInclusive<usize>> for Bounds {
     }
 }
 
+impl From<(RangeFrom<usize>, RangeTo<usize>)> for Bounds {
```

Comment:
I'd like to see documentation on this somewhere, it's not obvious from the implementation how it intended to be used.